Formula/beancount.rb

```diff
@@ -3,8 +3,8 @@ class Beancount < Formula
 
   desc "Double-entry accounting tool that works on plain text files"
   homepage "http://furius.ca/beancount/"
-  url "https://files.pythonhosted.org/packages/69/ea/ddf75e1e88aaba807bea3ac101b0e226ddc60832ecfe83ad05cc3a8fff4a/beancount-2.2.3.tar.gz"
-  sha256 "1554adfd773d12cb88fd7f4da67fcb608665a9bdedc7e44834e059d1b3a08e5d"
+  url "https://files.pythonhosted.org/packages/26/d6/9cb23ad3af828ebba10deda85dc28c0985acfbfdce3fb6c4b76ec69389d3/beancount-2.3.0.tar.gz"
+  sha256 "9a6d9692435007195aae29a20328fb11d0126ff03db66322a79a894b1f422712"
   head "https://bitbucket.org/blais/beancount", :using => :hg
```

Comment:
This says it has been migrated to "https://github.com/beancount/beancount" could you change that too